libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -161,6 +161,14 @@ export interface AccountBridge<T extends TransactionCommon> {
     parentAccount?: Account | null | undefined;
     transaction?: T | null | undefined;
   }): Promise<BigNumber>;
+  applyReconciliation?: (
+    account: Account,
+    updatedRaw: AccountRaw,
+    next: Account
+  ) => boolean;
+  toAccountRaw?: (account: Account, accountRaw: AccountRaw) => void;
+  fromAccountRaw?: (accountRaw: AccountRaw, account: Account) => void;
```

Comment:
just a wording choice but we should have a wording that indicates that this is doing a mutation and not a "pure" function.

maybe `assign*` ? assignFromAccountRaw, assignToAccountRaw

( wording choice ~ kinda like a `Object.assign` )

Reply:
Fixed. Thanks for the feedback.